build.sh

```diff
@@ -758,11 +823,19 @@ function build_micro_cpp()
 function build_cpp03()
 {
     copy_src_cpp_common
+    check_flatdata_available
     additional_defines_calculation "CPPModern"
+
     ##############################################################################
     # Generate files for srcCpp03
-
-    rtiddsgen_command="\"${rtiddsgen_executable}\" -language ${modern_cpp_lang_string} -unboundedSupport -replace -create typefiles -create makefiles -platform ${platform} -additionalHeaderFiles \"ThreadPriorities.h Parameter.h ParameterManager.h MessagingIF.h RTIDDSImpl.h perftest_cpp.h qos_string.h CpuMonitor.h PerftestTransport.h\" -additionalSourceFiles \"ThreadPriorities.cxx Parameter.cxx ParameterManager.cxx RTIDDSImpl.cxx CpuMonitor.cxx PerftestTransport.cxx\" -additionalDefines \"${additional_defines}\" ${rtiddsgen_extra_options} -d \"${modern_cpp_folder}\" \"${idl_location}/perftest.idl\""
+    rtiddsgen_command="\"${rtiddsgen_executable}\" -language ${modern_cpp_lang_string} \
```

Comment:
Check the code in master, I believe we reworked this, you will have to do some changes here.

Reply:
Do a merge with the latest changes from master.

Reply:
Forget about my last comment, but still, instead of listing the files, you can use the variable where the files are, like in cpp:

```
    additional_header_files=" \
        ThreadPriorities.h \
        Parameter.h \
        ParameterManager.h \
        MessagingIF.h \
        RTIDDSImpl.h \
        perftest_cpp.h \
        CpuMonitor.h \
        PerftestTransport.h \
        PerftestSecurity.h \
        Infrastructure_common.h \
        Infrastructure_micro.h"

    additional_source_files=" \
        ThreadPriorities.cxx \
        Parameter.cxx \
        ParameterManager.cxx \
        RTIDDSImpl.cxx \
        CpuMonitor.cxx \
        PerftestTransport.cxx \
        PerftestSecurity.cxx \
        Infrastructure_common.cxx \
        Infrastructure_micro.cxx"
```

Reply:
Done